src/Symfony/Component/Console/Helper/SymfonyQuestionHelper.php

```diff
@@ -29,6 +29,8 @@ class SymfonyQuestionHelper extends QuestionHelper
 {
     /**
      * {@inheritdoc}
+     *
+     * @deprecated since version 3.3, to be removed in 4.0
```

Comment:
Not sure about this.. it creates some nasty strikethroughs in IDE's, and the same method call will work in 4.0